src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -258,24 +265,31 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
                 // may well be both large and somewhere out on the
                 // internet.
                 Media::Imaging::BitmapImage image(imageUri);
-                brush.ImageSource(image);
+                _bgImageLayer.Source(image);
+                _bgImageLayer.HorizontalAlignment(HorizontalAlignment::Center);
+                _bgImageLayer.VerticalAlignment(VerticalAlignment::Center);
             }
 
             // Apply stretch and opacity settings
-            brush.Stretch(_settings.BackgroundImageStretchMode());
-            brush.Opacity(_settings.BackgroundImageOpacity());
+            _bgImageLayer.Stretch(_settings.BackgroundImageStretchMode());
+            _bgImageLayer.Opacity(_settings.BackgroundImageOpacity());
 
-            // Apply brush if it isn't already there
-            if (_root.Background() != brush)
-            {
-                _root.Background(brush);
-            }
         }
         else
+        {
+            _bgImageLayer.Source(nullptr);
+        }
+
+        if (!_settings.UseAcrylic())

```

Comment:
just to make sure: with this change, when the user has a background image set, and they aren't using acrylic, then their set `defaultBackground` will appear _below_ the image they've set, do any transparent parts of the image, correct?

Reply:
Yes